src/unix/notbsd/linux/mod.rs

```diff
@@ -165,6 +165,20 @@ s! {
         _pad: [::uint8_t; 48],
     }
 
+
+    pub struct sigevent {
+        pub sigev_value: sigval_t,
+        pub sigev_signo: ::c_int,
+        pub sigev_notify: ::c_int,
+        pub sigev_notify_function: ::dox::Option<extern fn(sigev_value: sigval_t)>,
```

Comment:
This may actually want to just be a `usize`, C has a habit of sometimes using sentinel values for function pointers that mean "invalid" or "do this other specific action"